apps/music/index.html

```diff
@@ -97,16 +97,16 @@ <h1 id="title-text" data-l10n-id="music">Music</h1>
           <div id="player-cover-caption">
             <div id="player-cover-artist" data-l10n-id="songArtist">Song Artist</div>
             <div id="player-cover-album" data-l10n-id="songAlbum">Song Album</div>
-            <button id="player-cover-share" class="player-cover-button" data-icon="share"></button>
+            <button id="player-cover-share" class="player-cover-button" data-icon="share" data-l10n-id="share-song"></button>
           </div>
           <div id="player-cover-buttons">
             <button id="player-album-repeat" class="player-cover-button" data-icon="repeat"></button>
-            <div id="player-album-rating">
-              <button class="rating-star" data-rating="1"></button>
-              <button class="rating-star" data-rating="2"></button>
-              <button class="rating-star" data-rating="3"></button>
-              <button class="rating-star" data-rating="4"></button>
-              <button class="rating-star" data-rating="5"></button>
+            <div id="player-album-rating" data-l10n-id="player-album-rating" role="region" aria-live="polite">
```

Comment:
I still think that it's best to have it as a radio button. I think it will be more confusing for the user to not be able to check more than one checkbox. Also we do not need aria-live here, afaik the rating is only changed by the user so the changes won't happen when the focus is not on the radiogroup.